app/pages/events/activity_register.php

```diff
@@ -0,0 +1,132 @@
+<?php
+restrict_access();
```

Comment:
I do not know if this is a good idea to have a separate entry page for activities... maybe we should make it so that if an activity is not standalone, we should redirect to the main event entry page.

I don't think all this work is lost though, since we can reuse it when we have standalone activity entry.

Also I think we can reuse a lot of code here, that you could extract into `eventUtils.php`

Reply:
Now that I read through this file, I realize it is a massive copy-paste from `event_register`. There are different opinions about copy-pasting code, but my personal take is this:
- copy pasting is fine when done on small chunks of code.
- copy pasting is fine when done on the same file.
- in all other cases, extract code instead of copy pasting.

Reply:
Got it, I tried to extract the code, not so easy : seems like a heavy task for just two pages, but I guess this is a good habit 🤝 Also makes the code easier to understand 👍

Reply:
For now let's keep just the event registration page available, but in the future this new feature will probably be useful. Still some work to be done to have the activities on the main page though.